.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,40 @@
+name: Bug report
+description: Create a report to help us improve QMK Firmware.
+title: "[Bug] "
+labels: ["bug", "help wanted"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Provide a general summary of the bug in the title above.
+  - type: textarea
+    attributes:
+      label: Describe the Bug
+      description: A clear and concise description of what the bug is.
+  - type: input
+    attributes:
+      label: Keyboard Used
+  - type: input
+    attributes:
+      label: Revision (if applicable)
+  - type: input
+    attributes:
+      label: Operating System
+  - type: textarea
+    attributes:
+      label: qmk doctor Output
+      description: Output from running the `qmk doctor` command.
+      render: shell
```

Comment:
```suggestion
```

Reply:
The `render: shell` just puts the input into a markdown code block. I thought this would be better for readability but if you don't think so just let me know and I will accept the change.

Reply:
The `render` attribute designates the language this is written in; `shell` implies some form of shell script (like bash), which is not what the `qmk doctor` output is. For this use case it's just plain text.